### PR TITLE
fix: replace TextInput and remove old package

### PR DESCRIPTION
**Pull request by burakukula**
In this PR:
 - apply copy button isDisabled option in TextInput
 - remove `label` from `BaseInput` 
 - replace old `TextInput` with the new one and remove the old one.

### CODE REVIEW

File: packages/components/forms/src/base-input/types.ts [hunk 1]
```diff
@@ -15,9 +15,9 @@ export interface BaseInputInternalProps extends CommonProps {
    */
   as?: 'input' | 'textarea'; // TODO: we can extend to select
   /**
-   * Label value is set as aria-label attribute in the input
+   * Aria-label attribute in the input
    */
-  label: string;
+  ariaLabel: string;
```
Comment (suevalov) [inline packages/components/forms/src/base-input/types.ts:20]: I think it should be optional, right?

Comment (burakukula) [reply packages/components/forms/src/base-input/types.ts:20]: hmm, I think we need to enforce it. If not we can have not accessible input used by the client (someone can decide that they don't need the label for some reason)?

File: packages/components/forms/src/base-input/BaseInput.tsx [hunk 2]
```diff
@@ -33,7 +33,7 @@ function _BaseInput<E extends React.ElementType = typeof DEFAULT_TAG>(
     isReadOnly,
     isInvalid,
     id,
-    label,
+    ariaLabel,
```
[the remark this answers is not in the document]
Comment (burakukula) [reply packages/components/forms/src/base-input/BaseInput.tsx:36]: I just wanted to make it required, not sure if that's the best way to do it?

File: packages/components/forms/src/base-input/types.ts (hunk 1 above)
Comment (suevalov) [reply packages/components/forms/src/base-input/types.ts:20]: I haven't seen it in other libraries. Do we still need it if an input is wrapped in a `label` tag?

File: packages/components/forms/src/base-input/BaseInput.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (massao) [reply packages/components/forms/src/base-input/BaseInput.tsx:36]: Ohh I mean, that if we don't pass it there, when we use this component we will be able to pass both `ariaLabel` and `aria-label` properties, not sure if this is the expected behaviour.

[the remark this answers is not in the document]
Comment (burakukula) [reply packages/components/forms/src/base-input/BaseInput.tsx:36]: Ah!, got it! It's not expected. Fixing. Thanks!

File: packages/components/forms/src/base-input/types.ts (hunk 1 above)
Comment (burakukula) [reply packages/components/forms/src/base-input/types.ts:20]: No, in that case, we wouldn’t need it. but what if user fogot about adding the label for some reason? I just thought we would have aria-label always, no matter if there is a label or not. 🤷‍♀️

**Review by suevalov — APPROVED** [review]

File: packages/components/forms/src/base-checkbox/types.ts [hunk 3]
```diff
@@ -27,4 +27,8 @@ export interface BaseCheckboxInternalProps
    * Additional props that are passed to the input element
    */
   inputProps?: Partial<ComponentProps<'input'>>;
+  /**
+   * Set label for checkbox components
+   */
+  label: string;
```
[the remark this answers is not in the document]
Comment (massao) [reply packages/components/forms/src/base-checkbox/types.ts]: I think we can remove this too, will do that on another PR, as we are using the children for the label, and the user can pass the `aria-label` if necessary.

**Pull request merged**